filebeat/scripts/docs_collector.py

```diff
@@ -74,7 +79,7 @@ def collect(beat_name):
     module_list_output += "include::modules-overview.asciidoc[]\n"
     for m, title in sorted(six.iteritems(modules_list)):
         module_list_output += "include::modules/" + m + ".asciidoc[]\n"
-
+    module_list_output += ":edit_url!:"
```

Comment:
I think that here it's only doing it at the end of the list. If that's what you want, that's fine, but I think clearing the variable at the end of each file where it's set (around line 67) might be better.